src/core/public/saved_objects/saved_objects_client.ts

```diff
@@ -446,9 +445,9 @@ export class SavedObjectsClient {
     const path = this.getPath([type, id]);
     const body = {
       attributes,
-      migrationVersion,
```

Comment:
were we sending `migrationVersion` to the route when it was never allowed according to the validation schema in the route? How is that possible?

Reply:
We just never were using this parameter when calling this API, so it was never included in the payload